modules/adding-tls-termination.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/add-tls-termination.adoc
+
+:_content-type: PROCEDURE
+[id="nw-adding-tls-termination_{context}"]
+= Adding the TLS termination on the AWS Load Balancer
+
+You can route the traffic for the domain, for example, `example.com` to pods of a `service`, for example, `example-service`, and add TLS termination on the AWS Load Balancer.
+
+.Prerequisites
+
+* You have an access to OpenShift CLI (oc).
+
+.Procedure
+
+. Install the Operator and create an instance of the `AWSLoadBalancerController` resource:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: AWSLoadBalancerController
+group: networking.olm.openshift.io/v1alpha1 <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: auto
+  ingressClass: tls-termination <3>
+----
+
+<1> Defines the API group of the `AWSLoadBalancerController` resource.
+<2> Defines the `aws-load-balancer-controller` instance.
+<3> Defines the name of the `ingressClass` reconciled by `aws-load-balancer-controller`. This `ingressClass` gets created if it is not present. You can add additional `ingressClass` values. The controller reconciles the `ingressClass` values if the `spec.controller` is set to `ingress.k8s.aws/alb`.
```

Comment:
Need nouns to follow e.g. `ingressClass`

Reply:
Addressed